install.ps1

```diff
@@ -509,7 +542,9 @@ function Install-AirgapTarball() {
 $STORAGE_URL = "https://storage.googleapis.com/rke2-ci-builds"
 $INSTALL_RKE2_GITHUB_URL = "https://github.com/rancher/rke2"
 $DEFAULT_TAR_PREFIX = "C:\usr\local"
+$INSTALL_RKE2_TAR_PREFIX = "C:\usr\local"
```

Comment:
Does this need to exist, as $DEFAULT_TAR_PREFIX is the same value above?

Reply:
its cause they both exist in the install.sh... dont disagree with you but im not sure it's my call

Reply:
Okay if they already are there, matching install.sh is fine.